lib/amperize.js

```diff
@@ -32,6 +32,8 @@ var DEFAULTS = {
   }
 };
 
+var called;
```

Comment:
The reason for declaring it out of the scope of `getImageSize()` is for testing. This way only, I can check the value of it, using `rewire`'s `__get__` functionality.